core-rust-bridge/src/main/java/com/radixdlt/rev2/Decimal.java

```diff
@@ -75,11 +75,31 @@
 import java.util.Objects;
 import org.bouncycastle.util.Arrays;
 
-/** Decimal represents a 192 bit representation of a fixed-scale decimal number. */
+/**
+ * Decimal represents a 192 bit representation of a fixed-scale decimal number. Note that the Java
+ * implementation is fairly basic and mostly acts as a container for node<->engine interop.
+ * Internally it uses an *unsigned* 192-bit integer representation, and only exposes a handful of
+ * operations that behave the same way for signed and unsigned numbers (e.g. add, sub).
+ */
 @SecurityCritical(SecurityKind.NUMERIC)
 public class Decimal implements Comparable<Decimal> {
 
-  public static final Decimal ZERO = Decimal.of(0L);
+  public static final Decimal MIN_VALUE =
+      Decimal.fromUnsignedFixedPointRepresentation(
+          UInt192.from(
+              new BigInteger("-3138550867693340381917894711603833208051177722232017256448")
+                  .toByteArray()));
```

Comment:
I see we're relying on the fact that `.toByteArray()`  for `UInt192` and `BigInteger` is identical. But we have the DecimalTests which seem to test this well, so I guess it's fine to rely on this.

Reply:
It doesn't always work for negative numbers (e.g. shorter than 24 bytes), but it does for MIN_VALUE. I'll add a comment to make that clear.